tutorials/physics/physics_introduction.rst

```diff
@@ -320,6 +320,13 @@ Contact monitoring via signals can be enabled via the :ref:`contact_monitor <cla
 property. See :ref:`RigidBody2D <class_RigidBody2D>` for the list of available
 signals.
 
+.. note:: Rigid bodies do not detect collisions with ``Area2D`` nodes. If you need a rigid body to detect collisions with ``Area2D``, consider using the ``body`` argument in the ``Area2D``  ``_on_body_entered(body)`` signal to emit a signal from the rigid body: 
+.. tabs::
```

Comment:
Not sure if this a good suggestion to give in the docs, but either way formatting needs to be improved like so:

```suggestion
.. note::

    Rigid bodies do not detect collisions with ``Area2D`` nodes. If you need a rigid body
    to detect collisions with ``Area2D``, consider using the ``body`` argument in the ``Area2D``
    ``_on_body_entered(body)`` signal to emit a signal from the rigid body: 

.. tabs::
```